bndtools.core/src/bndtools/editor/BndSourceEditorPage.java

```diff
@@ -189,4 +194,19 @@ private IDocument getDocument() {
     public Image getTitleImage() {
         return icon;
     }
+
+    @Override
+    protected void createActions() {
```

Comment:
Yes, I'm not sure how that code got tangled in this branch, but I think it wasn't quite working. I will create a separate PR for this and hopefully we can get the lingering issue sorted out :)